.clang-tidy

```diff
@@ -116,5 +121,11 @@ CheckOptions:
     value:           'false'
   - key:             google-readability-function-size.StatementThreshold
     value:           '800'
+  - key:             cppcoreguidelines-narrowing-conversions.WarnOnIntegerToFloatingPointNarrowingConversion
+    value:           'true'
+  - key:             readability-function-cognitive-complexity.DescribeBasicIncrements
+    value:           'false'
+  - key:             readability-function-cognitive-complexity.IgnoreMacros
+    value:           'true'
```

Comment:
Why enable this? Does it improve the report in some way?

Reply:
The content of a macro does not impact the perceived complexity of a function as far as lining should be concerned.

Reply:
Fine by me.